src/AppInstallerCLICore/Workflows/UpdateFlow.cpp

```diff
@@ -111,7 +111,9 @@ namespace AppInstaller::CLI::Workflow
 
             updateAllFoundUpdate = true;
 
-            updateContext << InstallPackageInstaller;
+            updateContext << 
+                ManageDependencies << 

```

Comment:
See how much cleaner the callsite is when you don't have to check the feature?  Also, you would have forgotten to check the feature here if you needed to.